### ISSUE TITLE
Bitcoind

### ISSUE THREAD

**Issue by yancyribbens** [blockchain-certificates/cert-issuer#135]
The Dockerfile is outdated and running an old version of bitcoind `Bitcoin Core RPC client version v0.13.2.0-g0d71914`.

**Comment by AnthonyRonning** [comment]
Yeah, definitely a good idea to update. Though there is a problem using p2sh addresses (which are now default) with cert-issuer. So if we do update Dockerfile before testing out https://github.com/blockchain-certificates/cert-issuer/pull/84/files more then we'd need to atleast update the docs to specify that legacy addresses need to be created. 

Might be a good idea to mention it anyways for `Dockerfile.web` since p2sh addresses are being created but having issues as specified here: https://community.blockcerts.org/t/unable-to-sign-transaction-testnet/1779/11

**Comment by yancyribbens** [comment]
Agree the docs would need to be updated to continue using the legacy addresses.  Also, we could script the address creating is i've done in this [PR](https://github.com/blockchain-certificates/cert-issuer/pull/136/files#diff-7a7ac2c71f39aadb8bb5d854fe4a88abR13)
